certified-connectors/Huddle US/apiDefinition.swagger.json

```diff
@@ -0,0 +1,896 @@
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/v2/folder/{folderId}/children": {
+      "get": {
+        "description": "Lists the child folders contained within the target folder",
+        "operationId": "GetChildFolders",
+        "parameters": [
+          {
+            "description": "The ID of the parent folder",
+            "in": "path",
+            "name": "folderId",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Folder"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "items": {
+                "$ref": "#/definitions/Folder"
+              },
+              "type": "array"
+            }
+          }
+        },
+        "summary": "Retrieve a list of all of the subfolders within a target folder",
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/v2/folder/{folderId}/upload": {
+      "post": {
+        "description": "Uploads a file to the target folder",
+        "operationId": "UploadFile",
+        "parameters": [
+          {
+            "description": "The workspace where the file will be uploaded",
+            "in": "query",
+            "name": "workspaceId",
+            "required": true,
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "GetWorkspaces",
+              "value-collection": "workspaces",
+              "value-path": "id",
+              "value-title": "title"
+            },
+            "x-ms-summary": "Workspace"
+          },
+          {
+            "description": "The folder to target for file upload",
+            "in": "path",
+            "name": "folderId",
+            "required": true,
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "GetWorkspacePaths",
+              "parameters": {
+                "workspaceId": {
+                  "parameter": "workspaceId"
+                }
+              },
+              "value-collection": "paths",
+              "value-path": "id",
+              "value-title": "path"
+            },
+            "x-ms-summary": "Target Folder"
+          },
+          {
+            "in": "body",
+            "name": "requestBody",
+            "schema": {
+              "$ref": "#/definitions/FileUpload"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "OK",
+            "schema": {
+              "properties": {
+                "file": {
+                  "$ref": "#/definitions/File"
+                }
+              },
+              "type": "object"
+            }
+          }
+        },
+        "summary": "Create and upload a new file to a target folder",
+        "x-ms-visibility": "important"
+      }
+    },
+    "/v2/poll/folder/{folderId}/upload": {},
```

Comment:
@seanspeaks Please remove all the empty action definitions.

Reply:
@sriyen-msft I can remove those, however this was the only way we were able to get the polling events/triggers to work. This could have changed since we first built the connector? I can try to pull down and remove and test, just wanted to flag in case I'm missing something.

Reply:
@sriyen-msft looks like you approved? I hadn't updated to remove the empty action... but I did notice that my commit is missing a second trigger that we had built (in my reforking I lost some updates :-/). 

Can this be merged? Am I allowed to do the merging?

I figure it's better for the merge to happen to start the process of getting the bulk of the Connector into the Preview region, then I can create a follow-up PR with the added Trigger. 

Just looking for some guidance on next steps here. Thanks!

Reply:
Cc @dougelliott in case I'm misunderstanding here

Reply:
@dmkuby Can you please review the comment and provide your input on how can an empty operation would help in polling triggers.

Reply:
Empty paths objects may be empty from the specification point of view, however, they do not make sense for connectors and they are ignored, so I do not think they would have an impact on other operations.
What is exactly not working with polling triggers without this empty path object? Is there an error or no polling calls?